src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -149,6 +157,8 @@ public override void _Ready()
         lineContainer = GetNode<Control>(LineContainerPath);
 
         nodeScene = GD.Load<PackedScene>("res://src/microbe_stage/editor/PatchMapNode.tscn");
+        populationIndicatorScene = GD.Load<PackedScene>(
```

Comment:
This is just a single node so for performance I don't think this should be made as a separate scene, but just instantiate the nodes with the right texture (load the texture here) in this class.